v2/src/features/UploadList/UploadList.tsx

```diff
@@ -52,11 +52,28 @@ const generateErrorPayload = (error: AxiosError | Error) => {
   return errorPayload;
 };
 
+const LOCAL_STORAGE_KEY = 'sugarshare.files';
+
```

Comment:
It would be good to abstract this away in a lib for good practice. This way we can mock it and add tests later.

Reply:
Methods would be get, set, sync, delete.
```ts
class Storage {
  constructor({ storageKey = 'sugarshare.files' } = {}) {}
  get()
  set()
  sync()
  delete()
}

Reply:
no classes please

Reply:
I'm not sure if I like the abstraction of the local storage, it is already "abstracted" it is a feature from the browser, I don't see much benefit of re-writing it

Reply:
unless we were using two types of storage as before, then maybe the abstraction would be beneficial but I think this is just making it a bit harder
```javascript
const Storage = () => {
 set: (key, value) =>  window.localStorage.set(key,value)
 get: (key) => window.localStorage.get(key)
}
```

I think it can make it ambiguous, like whats the difference between just use the localStorage on your component instead of importing it from a folder in your project

Reply:
Jesus once said: "Simplicity over complexity, complexity over complicatedness"

Reply:
This is becoming a coding style discussion

Reply:
Agree, my point was just this is a bit of over abstraction